unblob/file_utils.py

```diff
@@ -287,3 +288,11 @@ def read_until_past(file: io.BufferedIOBase, pattern: bytes):
             return file.tell()
         if next_byte not in pattern:
             return file.tell() - 1
+
+
+def valid_path(path: Path) -> bool:
```

Comment:
I would add some simple tests for this function.

Reply:
Added a test to test_file_utils with two unit tests (valid and invalid paths).